ieducar/Views/FinalResultController.php

```diff
@@ -114,8 +114,8 @@ public function beforeValidation()
         $this->report->addArg('texto_rodape', (string) $this->getRequest()->texto_rodape);
         $this->report->addArg('situacao', (int) $this->getRequest()->situacao_matricula_id);
         $this->report->addArg('dependencia', (int) $this->getRequest()->dependencia);
-        $this->report->addArg('criterio_aprovacao', (string) Portabilis_String_Utils::toLatin1($this->getRequest()->criterio_aprovacao));
-        $this->report->addArg('portaria_aprovacao_pontos', Portabilis_String_Utils::toLatin1((string) $GLOBALS['coreExt']['Config']->report->portaria_aprovacao_pontos));
+        $this->report->addArg('criterio_aprovacao', (string) Portabilis_String_Utils::toUtf8($this->getRequest()->criterio_aprovacao));
+        $this->report->addArg('portaria_aprovacao_pontos', Portabilis_String_Utils::toUtf8((string) $GLOBALS['coreExt']['Config']->report->portaria_aprovacao_pontos));
```

Comment:
O método `Portabilis_String_Utils::toUtf8` retorna a própria string e pode ser removido.

```suggestion
        $this->report->addArg('criterio_aprovacao', (string) $this->getRequest()->criterio_aprovacao);
        $this->report->addArg('portaria_aprovacao_pontos', (string) $GLOBALS['coreExt']['Config']->report->portaria_aprovacao_pontos);
```

Reply:
Obrigado @edersoares